src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceSeparatorErrors.java

```diff
@@ -1,9 +1,9 @@
-package  com.puppycrawl.   tools.checkstyle.checks.whitespace;
+package  com.puppycrawl.   tools.checkstyle.checks.whitespace.singlespaceseparator;
 
 import java.util.List;
 import  java.util.Vector;
 
-public class      InputSingleSpaceErrors  {
+public class      InputSingleSpaceSeparatorErrors  {
```

Comment:
@wate123 Maybe there is some confusion about what I am asking.
File should be renamed to `InputSingleSpaceSeparatorErrors.java`.
Class name must remain as `InputSingleSpaceErrors`.

My previous review was only pointing out that the file was not renamed.